src/app/components/text-input-field.tsx

```diff
@@ -119,6 +119,7 @@ export function TextInputField({
           ref={inputRef || ref}
           _disabled={{ bg: color('bg') }}
```

Comment:
Shouldn't `color` from @stacks/ui be replaced here?

Reply:
It should, thanks! I'll actually remove this change as it's not fully working, thanks!

I had opened a new ticket to apply the updated placeholder colours so I can include changing the disabled state colour also. 

https://github.com/leather-wallet/extension/issues/4345